src/NHibernate.Test/Async/Operations/MergeFixture.cs

```diff
@@ -189,9 +190,21 @@ private void Cleanup()
 		{
 			ClearCounts();
 
-			var root = new Node {Name = "root"};
-			var child = new Node {Name = "child"};
-			var grandchild = new Node {Name = "grandchild"};
+			var root = new Node
+			{
+				Created = RoundForDialect(DateTime.Now),
```

Comment:
Just fixing the trouble in test. It could be fixed at the type level by overriding `IsEqual` (and preferably setter and getter too), but this would be a possible breaking change for `timestamp` when used a as simple property, because this will cause it to use the dialect declared accuracy instead of the actual database accuracy.